conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -159,6 +159,7 @@ types:
           deprecated: optional<Documentation>
           markers: list<Type>
           tags: set<string>
+          errors: list<ErrorDefinition>
```

Comment:
I think we may want to use `set<TypeName>` here so that a single error definition may be used across several endpoints. We can validate that all error type-names can be found in the top level error definitions.